source/dubregistry/api.d

```diff
@@ -17,6 +17,8 @@ void registerDubRegistryWebApi(URLRouter router, DubRegistry registry)
 }
 
 interface IPackages {
+	Json querySearch(string q);
```

Comment:
I'd prefer a strongly typed (and using `@method`) signature like in #115:
`@method(HTTPMethod.GET) string[] search(string q, uint maxResults=5);` (not sure about the `maxResults` parameter)

Having `Json` in the D API is always prone to silent breakage and field access bugs.